samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -0,0 +1,515 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ * @brief Wi-Fi Raw Tx Packet sample
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(raw_tx_packet, CONFIG_LOG_DEFAULT_LEVEL);
+
+#include <nrfx_clock.h>
+#include <zephyr/kernel.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/init.h>
+
+#include <zephyr/net/net_if.h>
+#include <zephyr/net/wifi_mgmt.h>
+#include <zephyr/net/net_event.h>
+#include <zephyr/net/socket.h>
+#include <zephyr/drivers/gpio.h>
+
+#include "net_private.h"
+
+#define WIFI_SHELL_MODULE "wifi"
+
+#define WIFI_SHELL_MGMT_EVENTS (NET_EVENT_WIFI_CONNECT_RESULT |		\
+				NET_EVENT_WIFI_DISCONNECT_RESULT)
+
+#define MAX_SSID_LEN        32
+#define DHCP_TIMEOUT        70
+#define CONNECTION_TIMEOUT  100
+#define STATUS_POLLING_MS   300
+
+#define CONTINUOUS_MODE_TRANSMISSION 0
+#define FIXED_MODE_TRANSMISSION 1
+
+static struct net_mgmt_event_callback wifi_shell_mgmt_cb;
+static struct net_mgmt_event_callback net_shell_mgmt_cb;
+
+static struct {
+	const struct shell *sh;
+	union {
+		struct {
+			uint8_t connected	: 1;
+			uint8_t connect_result	: 1;
+			uint8_t disconnect_requested	: 1;
+			uint8_t _unused		: 5;
+		};
+		uint8_t all;
+	};
+} context;
+
+struct nrf_wifi_fmac_rawpkt_info {
```

Comment:
Why is this re-defined here? We should directly use the driver public header file.

Reply:
yes - if you can refer raw_tx_pkt_header from fmac_structs.h - kindly refer that structure